vendor/k8s.io/kubernetes/staging/src/k8s.io/kube-aggregator/pkg/controllers/status/available_controller.go

```diff
@@ -194,6 +223,28 @@ func (c *AvailableConditionController) sync(key string) error {
 	return err
 }
 
+func getDestinationHost(apiService *apiregistration.APIService, serviceLister v1listers.ServiceLister) string {
```

Comment:
how will this look like in 3.7? We get Walter's dialer code, right? Maybe add a Rebase 3.7 note here.